### DISCUSSION TITLE
Looking for a consultant to assist on Code updates and to create plugins for Chamilo

### DISCUSSION THREAD

**Discussion by etree2024** [chamilo/chamilo-lms#5194] [General]
Looking for a consultant to assist on Code updates and to create plugins for Chamilo.
We have an installation of chamilo that has need of extended functionality we are looking for someone to assist our our over extend ed staff deal with customer requirements. drop me a message. We are looking for those with experience in Symfony and Chamilo specifically as well as MySQL configuration and load testing. 

This is a paid position so we also need to know what your financial requirements are.

**Comment by ywarnier** [comment]
@etree2024 if it's for one-shot developments, don't hesitate to ask for quotes to the official providers: https://chamilo.org/providers
If it's not, please consider contributing the code to the project afterwards (it's as easy as doing a pull request and following the code conventions documented on the wiki).
Good luck!
